### DISCUSSION TITLE
Source Information to be pulled

### DISCUSSION THREAD

**Discussion by christianneureiter** [open-metadata/OpenMetadata#18890] [Ideas]
I did put a similar topic into "Issues", but do believe it fits better here.

I do believe, that there is some more "native" information which can be pulled from Source Systems into OM which bring great benefit. ER could be one thing, even though it needs to be somehow merged with Data Lineage BUT I am more thinking into the direction of "Source Descriptions". A lot of Application to offer that kind of information, which would a great benefit. 
SAP: Help via "F1" - for each and every object you do get a description via F1
Salesforce: Also here, there is a "Source Description" available.

This could be pulled in into OM as "Source Description" (Field Level) and the currently existing "Description" can be changed to "Business Description"

**Comment by Prajwal214** [comment]
We have a feature for ER Diagrams, but it is currently available only for Collate SaaS.
